test/propagate_constant_test.cpp

```diff
@@ -163,4 +164,64 @@ TEST_CASE(const_dot)
     EXPECT(m1 == m2);
 }
 
+TEST_CASE(literal_convert_scalar)
+{
+    migraphx::module m;
+
+    auto lit = m.add_literal(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {1}, {0}}));
+    auto t = m.add_instruction(
+        migraphx::make_op("convert",
+                          {{"target_type", migraphx::to_value(migraphx::shape::int32_type)}}),
+        lit);
+    m.add_return({t});
+
+    auto out_shape = m.get_output_shapes().back();
+    run_pass(m);
+
+    auto target_lit_out_shape = migraphx::shape{migraphx::shape::int32_type, {1}, {0}};
+    EXPECT(target_lit_out_shape == out_shape);
+}
```

Comment:
This also shouldn't compare output shapes.

Reply:
Done